configs/workflows-mt.yaml

```diff
@@ -0,0 +1,377 @@
+Workflows:
+  - Name: Sequencing Noninterleaved
+    Collection: omics_processing_set
+    Enabled: True
+    Filter Output Objects:
+    - Metagenome Raw Read 1
```

Comment:
Are we going to make new types for these (e.g. Metatranscriptome Raw Read...)

Reply:
We left this alone to minimize schema changes ahead of the berkeley refactor rollout. After that comes on in October and I'd to make this file type more generic like "Nucleotide Sequencing Read 1"